deployment/check-environment.sh

```diff
@@ -47,12 +47,28 @@ print("Attempting load of fortran-based modules... ", end='', flush=True)
 from wagl import singlefile_workflow
 print("✅")
 
-# The previous import of wagl should have initialised the filters.
+# TODO CHECK: The previous import of wagl should have initialised the filters.
 print("Attempting hdf5 blosc compression...", end='', flush=True)
 import h5py
+import hdf5plugin
 import tempfile
+
+# Test different forms of compression usage as per the hdf5plugin docs:
+# https://hdf5plugin.readthedocs.io/en/stable/usage.html
+# use "hdf5plugin.Blosc()" style instead of int code to prevent crashes
+print("Attempting hdf5 blosc compression...", end='', flush=True)
 f = h5py.File(tempfile.mktemp('-test.h5'),'w')
-dset = f.create_dataset("myData", (100, 100), compression=32001)
+dset = f.create_dataset("myData", (100, 100), compression=hdf5plugin.Blosc())
```

Comment:
I do recommend using wagl's compression config to generate the required kwargs.

```python
from wagl import hdf5
comp = hdf5.compression.H5CompressionFilter.BLOSC_ZSTANDARD
config = comp.config(chunks=(200, 300), aggression=9)
kwargs = config.dataset_compression_kwargs()
# print(kwargs)  to actually see what it comes back as
# {'chunks': (200, 300), 'shuffle': False, 'compression': 32001, 'compression_opts': (0, 0, 0, 0, 9, 1, 5)}

Reply:
this would then test that the plugin is available, and that the wagl config generator is doing it correctly.

Reply:
I've shrunk the script down to replace the old int codes with a single test case using the wagl's specific config in [b02d4d4](https://github.com/OpenDataCubePipelines/ard-pipeline/pull/69/commits/b02d4d415739926487b6e701c0b60b51aa98790d).

The `check-environment` script:
* Ran on my local machine (only failed on to find modtran, as expected)
* Passed on `gadi`.